requirements-dev.txt

```diff
@@ -1,4 +1,5 @@
 pre-commit
 black
 flake8
-django>=3.1.7
\ No newline at end of file
+django>=3.1.7
```

Comment:
Remove the django dependencies from this file... they aren't dev requirements. Dev requirements are related to testing or linting (or anything not related to building and running the project)